scripts/type_check_backend.sh

```diff
@@ -0,0 +1,8 @@
+cd "$(dirname ${0})/.."
+source scripts/helpers.sh
+
+pip_required
+
+pip install --upgrade pip
+pip install pyright
+pyright gradio/context.py  gradio/blocks.py
```

Comment:
Nice! Let's file issues for adding typing to the other modules?

Reply:
I'll just go through the rest in the next few days ;)